go.sum

```diff
@@ -240,6 +240,8 @@ github.com/flyteorg/flyteidl v0.21.4 h1:gtJK5rX2ydLAo2xLRHHznOSLuLHrRRdXDbpEAlxl
 github.com/flyteorg/flyteidl v0.21.4/go.mod h1:576W2ViEyjTpT+kEVHAGbrTP3HARNUZ/eCwrNPmdx9U=
 github.com/flyteorg/flyteplugins v0.7.1 h1:YdCEQtdPeol7u6LkopGTIfPLAhy3KcclQa+DZFauK8w=
 github.com/flyteorg/flyteplugins v0.7.1/go.mod h1:kOiuXk1ddIEVSPoHcc4kBfVQcLuyf8jw3vWJT2Was90=
+github.com/flyteorg/flyteplugins v0.7.2 h1:WLftq5+/RYt3HLc/mjRprqexot3brlMJHMH1yS7jee4=
```

Comment:
Can you run `go mod tidy` to clean up the extra imports here...

Reply:
done!